test/extractors/IndividualBeneficiaryExtractorSpec.scala

```diff
@@ -160,7 +161,7 @@ class IndividualBeneficiaryExtractorSpec extends SpecBase {
                 incomeDiscretionYesNo = Some(false),
```

Comment:
It would be good to use mentalCapacityYesNo = None / DontKnow in some of the cases